sphinx/locale/__init__.py

```diff
@@ -229,11 +229,17 @@ def lazy_gettext(string):
 if PY3:
     def _(message):
         # type: (unicode) -> unicode
-        return translators['sphinx'].gettext(message)
+        try:
+            return translators['sphinx'].gettext(message)
+        except KeyError:
+            return message
```

Comment:
This is needed for `Config.convert_overrides()`. It is called both before and after `_init_i18n()`. The KeyError might be raised only if user gives invalid values to pre-init config values.